amplify/src/main/java/com/github/stkent/amplify/tracking/GenericSettings.java

```diff
@@ -79,4 +79,8 @@ public T getEventValue(@NonNull final IEvent event) {
         return null;
     }
 
+    public boolean hasEventValue(@NonNull final IEvent event) {
+        return sharedPreferences.contains(event.getTrackingKey());
```

Comment:
Can add an abstract method to GenericSettings to request a prefix from each subclass?